packages/gensx/tests/commands/environment/list.test.ts

```diff
@@ -24,27 +38,40 @@ vi.mock("../../../src/models/projects.js", () => ({
   checkProjectExists: vi.fn(),
 }));
 
-vi.mock("../../../src/utils/project-config.js", () => ({
-  readProjectConfig: vi.fn(),
-}));
+// Set up and tear down the test environment
+beforeAll(async () => {
+  const setup = await setupTestEnvironment("list-test");
+  tempDir = setup.tempDir;
+  origCwd = setup.origCwd;
+  origConfigDir = setup.origConfigDir;
+});
+
+afterAll(async () => {
+  await cleanupTestEnvironment(tempDir, origCwd, origConfigDir);
+});
 
-// Mock console output
-const originalConsoleInfo = console.info;
 beforeEach(() => {
-  console.info = vi.fn();
+  // Set working directory to our test project
+  process.cwd = vi.fn().mockReturnValue(path.join(tempDir, "project"));
 });
 
-afterEach(() => {
-  console.info = originalConsoleInfo;
+afterEach(async () => {
   vi.resetAllMocks();
+  await cleanupProjectFiles(tempDir);
 });
```

Comment:
I think all this test setup could be moved into a setup.ts file. https://vitest.dev/config/#setupfiles

Reply:
fair, i can do that. i refactored a decent bit of the code into a helpers fule but agreed the shared setup would be best.

Reply:
actually i'll do this refactoring with my next PR that upgrades everything else to ink; otherwise i'll end up refactoring twice.